lib/gateways/sessions.rb

```diff
@@ -2,16 +2,23 @@ module Gateways
   class Sessions
     MAXIMUM_RESULTS_COUNT = 500
 
-    def initialize(ips:)
-      @organisation_ips = ips
+    def initialize(ip_filter:)
+      @ip_filter = ip_filter
     end
 
     def search(username: nil, ips: nil)
-      results = Session
-        .where(siteIP: organisation_ips)
-        .where('start >= ?', 2.weeks.ago)
-        .order(start: :desc)
-        .limit(MAXIMUM_RESULTS_COUNT)
+      results = if ip_filter
```

Comment:
I believe this allows an organisation without an IP addresses to search all logs, due to the `if` statement being too permissive.

```rb
!![]
=> true
```

Reply:
retracting this, as I didn't realise an empty array is truthy